<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DmpClientConnectDialog</class>
 <widget class="QDialog" name="DmpClientConnectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>144</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>110</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>341</width>
     <height>104</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="NameLabel">
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="NameEdit">
      <property name="placeholderText">
       <string>Your screen name</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="PortLabel">
      <property name="text">
       <string>Port</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="PortEdit">
      <property name="placeholderText">
       <string>The server port</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="HostLabel">
      <property name="text">
       <string>Host</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="HostEdit">
      <property name="placeholderText">
       <string>The server hostname</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>NameEdit</tabstop>
  <tabstop>HostEdit</tabstop>
  <tabstop>PortEdit</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DmpClientConnectDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DmpClientConnectDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>HostEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>DmpClientConnectDialog</receiver>
   <slot>inputChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>NameEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>DmpClientConnectDialog</receiver>
   <slot>inputChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PortEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>DmpClientConnectDialog</receiver>
   <slot>inputChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>inputChanged()</slot>
 </slots>
</ui>
